pipelines/pyproject.toml

```diff
@@ -1,30 +1,37 @@
-[project]
-name = "turbo-template"
-authors = [
-    {name = "Example User", email = "user@example.com"},
-]
+[tool.poetry]
+name = "pipelines"
+version = "0.1.0"
+authors = ["Example User <user@example.com>"]
 description = "Turbo Template"
 readme = "README.md"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.9",
 ]
-requires-python = ">=3.7"
-dynamic = ["version"]
-dependencies = [
-    "kfp == 1.8.21",
+packages = [
+    { include = "pipelines", from = "src" },
 ]
 
-[project.optional-dependencies]
-tests = [
-    "google-cloud-bigquery == 2.30.0",
-    "pytest >= 7.3.1,<8.0.0",
-]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.11"
+Jinja2 = ">=3.0.1,<4.0.0"
+google-cloud-aiplatform = "1.24.1"
+google-cloud-pipeline-components = "1.0.42"
+pytest = "^7.3.1"
```

Comment:
remove `pytest` from dependencies (as it is already in the dev dependencies below)

Reply:
149ab44